src/index.d.ts

```diff
@@ -361,8 +361,20 @@ declare namespace RamdaAdjunct {
         /**
          * Merge objects under corresponding paths.
          */
-        mergePaths(ps: Array<Array<string | number>>, obj: object): object;
-        mergePaths(ps: Array<Array<string | number>>): (obj: object) => object;
+        mergePaths(paths: Array<Array<string | number>>, obj: object): object;
+        mergePaths(paths: Array<Array<string | number>>): (obj: object) => object;
+
+        /**
+         * Merges user provided data (subject) with some object (target) inside other object under corresponding property.
+         */
+        mergeProp(p: string, obj: object): object;
+        mergeProp(p: string): (obj: object) => object;
+
+        /**
+         * Merges user provided data (subject) with some object (target) inside other object under corresponding path.
```

Comment:
Provide full description here:

Create a new object with the own properties of the object under the `path` merged with the own properties of the provided `source`. If a key exists in both objects, the value from the `source` object will be used.